### ISSUE TITLE
Questions about the Settings

### ISSUE THREAD

**Issue by GiraPomba** [SlavicPotato/ied-dev#36]
Hey, a couple of questions and a possible issue.

Q: I can't find the setting for "unpaused menus" anymore. There seems to be some changes recently about this, so the option may be back.
Q: What does Interactive Interface>Enable weapons do?

Issue: In Settings>UI, the option "Exit on last window close" doesn't seem to work anymore after disabling it once.
Using latest preview build.

**Comment by SlavicPotato** [comment]
> With latest preview, I meant the 1.7.0, without the experimental loader thing.

Many more changes were made, github just happens to show the most recent commit if the description is left empty. In the top right corner, click compare and select the first tag to see all new commits from the previous version.

**Comment by GiraPomba** [comment]
Yeah, I didn't check the build from 3 days ago before posting, my mistake. 
I installed the newest version and it doesn't have the Interactive Interface anymore.

Anyway, about the option, "Physics debug renderer" was enabled under View. I just don't remember enabling this setting after messing with "Exit on last window close". I couldn't disable the renderer while "Exit on last window close" was on as it was closing the UI before unchecking, but I now I got the options working again properly. 

Thanks.
